cli/build.rs

```diff
@@ -30,9 +30,6 @@ fn main() {
         target_os = "dragonfly",
     ))]
     println!("cargo:rustc-link-arg=-Wl,--dynamic-list=cli/dynamic-symbols.txt");
-
-    #[cfg(any(target_os = "macos", target_os = "ios"))]
-    println!("cargo:rustc-link-arg=-Wl,-exported_symbols_list,cli/dynamic-symbols-darwin.txt");
```

Comment:
do we still need the `dynamic-symbols-darwin.txt` file?

Reply:
not currently, but I sort of wanted to take a deeper dive and figure out why it doesn't work only for macOS, and see if I can fix it probably tomorrow.

Reply:
Ok, sounds good.